index.php

```diff
@@ -0,0 +1,25 @@
+<?php
+$db = new PDO(
+    'mysql:host=DB;dbname=collections',   //DSN
+    'root',                    // Database username
+    'password'                 //Database password
+);
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+$query = $db->prepare("SELECT `location`, `sublocation`, `attraction`,`info`, `photo`, `rating`  FROM `USA`");
+$result = $query->execute();
+
+$location=[];
```

Comment:
not used yet

Reply:
Remove if not needed - we never want to have unused code in main